content/guides/gatsby/adding-tina/overview.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Overview
+---
+
+This guide will show you how to set up Tina on a Gatsby site. Since Gatsby let's you pull contentin from multiple data sources, these docs will only cover adding to Tina to your site and creating forms for content only stored in memory.
```

Comment:
```suggestion
This guide will show you how to set up Tina on a Gatsby site. Since Gatsby lets you pull content in from multiple data sources, these docs will only cover adding to Tina to your site and creating forms for content stored in memory.
```